wp-content/civi-extensions/goonjcustom/Civi/GoonjActivitiesService.php

```diff
@@ -580,4 +588,94 @@ public static function createActivityForGoonjActivityCollectionCamp(string $op,
     }
   }
 
+  /**
+   *
+   */
+  public static function sendActivityLogisticsEmail($collectionCamp) {
+    try {
+      $campId = $collectionCamp['id'];
+      $activityCode = $collectionCamp['title'];
+      $activityOffice = $collectionCamp['Goonj_Activities.Goonj_Office'];
+      $activityAddress = $collectionCamp['Goonj_Activities.Where_do_you_wish_to_organise_the_activity_'];
+      $activityAttendedById = $collectionCamp['Logistics_Coordination.Camp_to_be_attended_by'];
+      $logisticEmailSent = $collectionCamp['Logistics_Coordination.Email_Sent'];
+      $outcomeFormLink = $collectionCamp['Goonj_Activities.Select_Goonj_POC_Attendee_Outcome_Form'];
+
+      $startDate = new \DateTime($collectionCamp['Goonj_Activities.Start_Date']);
+
+      $today = new \DateTimeImmutable();
+      $endOfToday = $today->setTime(23, 59, 59);
+
+      if (!$logisticEmailSent && $startDate <= $endOfToday) {
+        $campAttendedBy = Contact::get(FALSE)
+          ->addSelect('email.email', 'display_name')
+          ->addJoin('Email AS email', 'LEFT')
+          ->addWhere('id', '=', $activityAttendedById)
+          ->execute()->single();
+
+        $attendeeEmail = $campAttendedBy['email.email'];
+        $attendeeName = $campAttendedBy['display_name'];
+
+        if (!$attendeeEmail) {
+          throw new \Exception('Attendee email missing');
+        }
+
+        $mailParams = [
+          'subject' => 'Goonj Activity Notification: ' . $activityCode . ' at ' . $activityAddress,
+          'from' => self::getFromAddress(),
```

Comment:
getFromAddress dont use this function here we are using it from helperservice file please check to out